packages/ui5-library-inquirer/src/index.ts

```diff
@@ -33,14 +59,31 @@ async function getPrompts(promptOptions?: UI5LibraryPromptOptions): Promise<Ques
  * @returns the prompt answers
  */
 async function prompt(promptOptions?: UI5LibraryPromptOptions, adapter?: InquirerAdapter): Promise<UI5LibraryAnswers> {
-    const ui5LibPrompts = await exports.getPrompts(promptOptions);
+    const ui5LibPrompts = await getPrompts(promptOptions);
     const pm = adapter ? adapter.promptModule : inquirer;
 
     if (pm && promptOptions?.useAutocomplete) {
         pm.registerPrompt('autocomplete', autocomplete);
     }
 
-    return adapter ? adapter.prompt(ui5LibPrompts) : inquirer.prompt(ui5LibPrompts);
+    const answers: UI5LibraryAnswers = adapter
+        ? await adapter.prompt(ui5LibPrompts)
+        : await inquirer.prompt(ui5LibPrompts);
+
+    // If a UI5 version was selected, resolve it to the nearest available npm version
```

Comment:
```suggestion
    // If a UI5 version was selected, resolve it to the nearest available npm version
    // Flow is: 
    // 1. User selects version from ui5 CDN supported versions 
    // 2. Selected version is used to get the nearest ui5 version from npm as ui5.yaml is using framework rather than ui5 cdn.
```